enterprise_access/apps/subsidy_access_policy/tests/test_models.py

```diff
@@ -483,6 +508,7 @@ def test_learner_enrollment_cap_policy_can_redeem(
             # Expected can_redeem result: False
             'policy_is_active': True,
             'catalog_contains_content': True,
+            'enroll_by_date': '2099-01-01T00:00:00Z',
```

Comment:
[nit/question/sanity check] I might just not be seeing it, but is there a test case permutation where `enroll_by_date` is missing?

Reply:
ah, nope, can add one